curl_cffi/requests/websockets.py

```diff
@@ -33,30 +64,269 @@ class WsCloseCode(IntEnum):
 
 
 class WebSocketError(CurlError):
```

Comment:
SessionClosed is a subclass of `RequestsError`, if this also subclasses from RequestsError, it would be easier to catch all error in websockets.

Reply:
Ah makes sense, I can make a new WebSocketClosed exception